pkg/api/exp/injection.go

```diff
@@ -0,0 +1,91 @@
+package exp
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+const (
+	// Deprecated: Dedicated field since v1beta3.
+	InjectionDisableMetadataEnrichmentKey = FFPrefix + "disable-metadata-enrichment"
+	// Deprecated: Dedicated field since v1beta3.
+	InjectionMetadataEnrichmentKey = FFPrefix + "metadata-enrichment"
+
+	InjectionIgnoreUnknownStateKey    = FFPrefix + "ignore-unknown-state"
+	InjectionIgnoredNamespacesKey     = FFPrefix + "ignored-namespaces"
+	InjectionAutomaticKey             = FFPrefix + "automatic-injection"
+	InjectionLabelVersionDetectionKey = FFPrefix + "label-version-detection"
+	InjectionFailurePolicyKey         = FFPrefix + "injection-failure-policy"
+	InjectionSeccompKey               = FFPrefix + "init-container-seccomp-profile"
+	InjectionEnforcementModeKey       = FFPrefix + "enforcement-mode"
+)
+
+// Deprecated: Dedicated field since v1beta3.
+func (ff *FeatureFlags) DisableMetadataEnrichment() bool {
+	return ff.getDisableFlagWithDeprecatedAnnotation(InjectionMetadataEnrichmentKey, InjectionDisableMetadataEnrichmentKey)
+}
+
+// IgnoreUnknownState is a feature flag that makes the operator inject into applications even when the dynakube is in an UNKNOWN state,
+// this may cause extra host to appear in the tenant for each process.
+func (ff *FeatureFlags) IgnoreUnknownState() bool {
+	return ff.getFeatureFlagBool(InjectionIgnoreUnknownStateKey, false)
+}
+
+// IsInjectionAutomatic controls OneAgent is injected to pods in selected namespaces automatically ("automatic-injection=true" or flag not set)
+// or if pods need to be opted-in one by one ("automatic-injection=false").
+func (ff *FeatureFlags) IsInjectionAutomatic() bool {
```

Comment:
```suggestion
func (ff *FeatureFlags) IsAutomaticInjection() bool {
```

Reply:
Reads better IMO

Reply:
but i like original version more, it's easy to read to me 🙈

Reply:
which is the original now 😅

Reply:
I don't have a strong opinion there - I can live with both :)

Reply:
based on go.doc:
>**is** **injected** to pods in selected namespaces **automatically**